### ISSUE TITLE
To batch or not to batch

### ISSUE THREAD

**Issue by eri-trabiccolo** [gocodebox/lifterlms-rest#212]
https://make.wordpress.org/core/2020/11/20/rest-api-batch-framework-in-wordpress-5-6/

**Comment by thomasplevy** [comment]
Yes, we should. This looks good to me. I don't really see any reason not to implement this for various endpoints, do you?

**Comment by eri-trabiccolo** [comment]
No I don't but I was opening this from my smartphone and I wanted to be funny and concise at the same time, that's the reason for the title.

**Comment by thomasplevy** [comment]
![kevin](https://user-images.githubusercontent.com/1290739/100119929-42dab400-2e2c-11eb-9b80-4f0572358298.gif)
